### DISCUSSION TITLE
how should I fork, add my less and css changes, compile and build ant design to be used as a custom package?

### DISCUSSION THREAD

**Discussion by beumsk** [ant-design/ant-design#37781] [Q&A]
Hello,

I would like to be able to create a package for multiple project that will be an adjusted version of AntD.
I didn't find a proper way to do it so far. I tried forking this repo, modify the less variables but I don't see how to manage the next steps.
Indeed I need a way to compile the less files and a way to build everything to be packaged.
I thought there would be something in the package.json for those but so far I wasn't lucky.

Any suggestions would be really appreciated 🙌

**Comment by zombieJ** [comment]
You can ref dev guide here: https://ant.design/docs/react/contributing#Development-Workflow
